OracleTuxedo/ART_SALT_TMA/dockerfiles/README.md

```diff
@@ -0,0 +1,21 @@
+# Introduction
+This folder contains the information and examples of how to use [Tuxedo](http://oracle.com/tuxedo) with [Docker](https://www.docker.com/).
+
+## Contents
+It is based on the WebLogic Server dockerization (is that even a word?) done by Bruno Borges.
+
+## To use
+1. Into an empty directory:
+  1. Download the Tuxedo TMA 12.2.2 Linux 64 bit installer from OTN
+  2. Download TuxedoARTTMA.zip from this github directory
+  3. Download the latest Tuxedo rolling patch from My Oracle Support, not less than RP003
+2. Unzip TuxedoARTTMA.zip
+3. Execute buildDockerImage.sh -p <RP Name>
+
+You can then start the image in a new container with:
+
+docker run --privileged -d -p 11122:22 -p 18080:8080 -e DISPLAY=bej301699.cn.oracle.com:1.0 -v /tmp/.X11-unix:/tmp/.X11-unix -h arthost --name tuxedoartrttma oracle/tuxedoartrttma:12.2.2 /sbin/init
```

Comment:
There are a few problems here: 

- We cannot publish a Dockerfile that requires `--privileged` access for a running container. This is against Oracle security policy guidelines. This needs to be removed.
- What is the purpose of the X11 volume mount? Also, how would this work on non-Linux based platforms, e.g. Docker for Windows or Mac?
- You can't run `/sbin/init` as we don't want to support an entire `init` system. You should be relying on an `ENTRYPOINT` or `CMD` directive to actually run the process required.